db-scripts/src/main/resources/migration.sql

```diff
@@ -543,8 +543,8 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.7.2";
 ##version: 2.7.3
 CREATE TABLE `cancer_study_tags` (
   `CANCER_STUDY_ID` int(11) NOT NULL,
-  `TAGS` text,
-  UNIQUE (`CANCER_STUDY_ID`),
+  `TAGS` text NOT NULL,
```

Comment:
Indeed, that's a bit tricky, because it is fixing something that is already wrong in rc. I can just add a version 2.7.4 that fixes the changes for table cancer_study_tags, or assume that people in rc using the schema from 2.7.3 will fix its code (since jumping from 2.7.2 to 2.7.4 in a new release might be a bit weird). @sheridancbio @jjgao @n1zea144 @inodb What would you prefer to do?

Reply:
@oplantalech 	2.7.3 should be ok, we haven't applied this migration yet anywhere. Sometimes we do apply rc's migration to our production databases but this one hasn't been done yet.